src/statement.rs

```diff
@@ -420,6 +435,17 @@ pub(crate) struct Statement {
     pub(crate) proof_end: usize,
 }
 
+pub(crate) const DUMMY_STATEMENT: Statement = Statement {
+    stype: StatementType::Invalid,
+    span: Span::NULL,
+    label: Span::NULL,
+    group: NO_STATEMENT,
+    group_end: NO_STATEMENT,
+    math_start: 0,
+    proof_start: 0,
+    proof_end: 0,
+};
```

Comment:
It's not used for the `$t` comment diagnostics, these actually do use the `$t` statement address. Rather, it is used for "line too long" diagnostics, which use a completely independent parse from the statement list so it would be more difficult to determine what statement if any contributed the characters in this long line.